de.metas.swat/de.metas.swat.base/src/main/sql/postgresql/system/40-sw01_swat/5496860_sys_gh4220_LotNoLock_Product_NotMandatory.sql

```diff
@@ -0,0 +1,5 @@
+-- 2018-06-29T17:27:07.163
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+UPDATE AD_Column SET IsMandatory='N',Updated=TO_TIMESTAMP('2018-06-29 17:27:07','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Column_ID=559563
```

Comment:
AFAIU we also need the DDL to make the physical column not-mandatory

Reply:
Done